data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1477 @@
+				goto next
+			label uhoh
+			`Though you don't display it outwardly, you realize that must be the dockworker you almost attacked on Lodestone. Better choose your words carefully, or things might get ugly.`
+			choice
+			`	"No...why do you ask? And what does he want?"`
+				goto oops
+			`	"I know I acted out of line on Lodestone - I promise it won't happen again."`
+				goto glare
+			label oops
+			`"Justice," he says as he procures a hidden blaster pistol from his shirt. Before you can realize your error, he suddenly shoots you several times in the stomach. You have just enough life in you to watch him be carted off by other Navy personnel before your vision is obscured by blackness.`
+			`You are dead. Let this be a lesson to you - if you act like an asshole, the galaxy tends to respond in kind.`
+				die
+			label glare
+			`"I'm gonna hold you to that, Blue Streak," he says, referencing your uniform. "Just know that not all of us are that forgiving."`
+				goto end
+			label next
+			branch help
+				has "rescue"
+			`	"No real reason. Just curious."`
+			`With some slight confusion, you just shrug and continue to wait while the rest of the mechanics are loaded aboard the transports.`
+				goto end
+			label help
+			apply
+				set "plan"
+				log "people" "Henry Radcliff" `An associate of Lucas Hadley, Radcliff and Hadley go way back. The two are practically brothers, and have vowed to keep an eye out for each other if anything went sideways. He also vows to be a private informant in case he sees anything sketchy.`
+			`	His expression suddenly softens. "The name's Henry Radcliff. I need to talk to you in private, if you don't mind." You nod, then turn to the other auxiliary captains. "I'll handle this one," you say to them.`
+			`Back aboard your ship, you lock the doors to your quarters before you and Radcliff sit down. "I'm glad I found you, Lieutenant," he begins. "Ever since we both started working for the Syndicate, Lucas and I made a promise to watch each others' backs. And it's my understanding that you made a promise to him, as well?"`
+			`	You nod. "When I learned that one of our officers had threatened Hadley, I wasn't about to let something like that slide."`
+			`	"And I thank you for that - there's no way I could bring that to my boss, and anyone else would just ignore me. But with you here, I think we have a real chance at getting Hadley and his family to safety."`
+			`	"So what's the plan?"`
+			`Henry procures a small holo-projector. "When the bombings happened, Lucas and I knew we had to come up with a contingency in case things got ugly. After considering our options, we decided to select this place." The projector highlights Hermes in the Capella System.`
+			`	"Hermes?" you say, with a sly grin. "That makes perfect sense. The planet's got orbital bombardment stations trained on the planet around the clock - if someone tried to tail you there, it's almost certain they'd be atomized by the relays."`
```

Comment:
... Yes, because the orbital bombardment system just fires at anyone...

The point of the orbital bombardment system is to clean an area before anything infectious spreads.  It's not there for planetary defense.  Tailing someone there would be a bit obvious, but so would having a family living there.  It's research stations mostly, and people living there need a government permit.